docs/wallet_API.md

```diff
@@ -83,19 +83,19 @@ const wallet = new BeaconWallet(options);
 
 The necessary bare minimum to instantiate the wallet is an object with a `name` property that contains the name of your dapp. However, the Beacon wallet allows you to customize your dapp responses to different events. In the example above, instead of getting the default Beacon pop-up after the user connects the wallet, it will display the available data in the console. You can use whatever solution you prefer for feedback. You can find a list of all the default handlers [in the beacon-sdk Github repo](https://github.com/airgap-it/beacon-sdk/blob/master/src/events.ts).
 
-> Note: if you want to use the Kukai wallet for testing on ithacanet, you must use the optional property `preferredNetwork` and set it to `ithacanet`, otherwise the mainnet version of the Kukai wallet will open.
+> Note: if you want to use the Kukai wallet for testing on jakartanet, you must use the optional property `preferredNetwork` and set it to `jakartanet`, otherwise the mainnet version of the Kukai wallet will open.
 
 The Beacon wallet requires an extra step to set up the network to connect to and the permissions:
 
 ```js
 await wallet.requestPermissions({
   network: {
-    type: 'mainnet' | 'hangzhounet' | 'ithacanet' | 'custom',
+    type: 'mainnet' | 'hangzhounet' | 'jakartanet' | 'custom',
```

Comment:
Consider removing hangzhounet instead of ithacanet

Reply:
ooops

Reply:
fixed